CMakeLists.txt

```diff
@@ -352,7 +357,7 @@ else ()
   set (_configure_date "Unknown")
 endif()
 include_directories (${CMAKE_BINARY_DIR})
-configure_file (build/version_string.ver.in version_string.ver @ONLY)
+configure_file (${CMAKE_CURRENT_LIST_DIR}/build/version_string.ver.in version_string.ver @ONLY)
```

Comment:
This I think should be ${CMAKE_CURRENT_BINARY_DIR}, if anything. CMAKE_CURRENT_LIST_DIR will clutter up the input tree with an extra file during an out-of-tree build.